**source: [Baker, J., Bradley, B., & Stafford, P. (2021). Seismic hazard and risk analysis. Cambridge University Press., Page 63](https://books.google.co.id/books?hl=en&lr=&id=preiEAAAQBAJ&oi=fnd&pg=PP1&ots=aTA17-0onZ&sig=7EvIKh8fJMcwqXfvVq_6-_x-oOM&redir_esc=y#v=onepage&q&f=false)**

A simple fault has some initial information as follows:

1. Strike-slip mechanism (dip = 90 degree),
2. Fault length of 40 Km,
3. Seismogenic depth of 11 Km,
4. Occurrence rate of M = 6.7 is 0.02
5. Fault slip-rate is 10 mm/year or 0.01 m/year

Keep in mind, the rupture rate is calculated by using the following equation

$\lambda(rup) = \frac {\dot{\bar u}} {\bar u}$

where $\dot{\bar u}$ is slip rate of fault (in mm/year or m/year) and $\bar u$ is average displacement. The slip rate and average displacement are obtained from geodetic data. However, the average displacement can be calculated by using seismic moment (geologic) approach or [Wells-Coppersmith's (1994) relationship](https://www.researchgate.net/publication/215755871_New_Empirical_Relationships_among_Magnitude_Rupture_Length_Rupture_Width_Rupture_Area_and_Surface_Displacement).

## The Seismic Moment Approach

The seismic moment is described through the following equation

$M_o = \mu A \bar {u}$

where the $M_o$ is seismic moment in $Nm$, $\mu$ is rigidity that equivalent to $3.3 x 10^{10} N/m^2$, $A$ is ruptre area in $m^2$, and $\bar {u}$ is average displacement in $mm$ or $m$. Then, to calculate the average displacement, the equation become

$\bar {u} = \frac {M_o} {\mu A}$

Next, we need the rupture area. This value product of length of fault and width of fault. In this case, the width of fault is not be provided. Then, the width of fault is calculated by using [Wells-Coppersmith's (1994) relationship](https://www.researchgate.net/publication/215755871_New_Empirical_Relationships_among_Magnitude_Rupture_Length_Rupture_Width_Rupture_Area_and_Surface_Displacement) for the strike-slip mechanism as follows

$log_{10} W = -0.76 + (0.27 M)$

$W = 10^{-0.76 + (0.27 M)}$ in $Km$

In [93]:
# initial information
dip = 90
fault_length = 40
fault_depth = 11
occurrence_rate_obs = 0.02
fault_slip_rate = 0.01
magnitude = 6.7

In [94]:
# Calculate the width of fault
fault_width = 10**(-0.76 + (0.27 * magnitude))
fault_width

11.19437883467152

Now, we have width of fault approximately of 11.2 Km. Then, the rupture area can be calculated

$A = L_{fault} x W_{fault}$

In [95]:
A = fault_length * fault_width
A # in km^2

447.7751533868608

Next, the $\bar u$ can be calculated where the $M_o$ uses the relationship of $M_o = 10^{M + 6.03}$

In [96]:
A_meter = A * 1000000 # convert to meter-square

In [97]:
mu = 3.3 * (10**10)
Mo = 10**(1.5*(magnitude + 6.03))
u_bar = Mo / (mu * A_meter)
u_bar # in meter

0.8422210055163545

We obtained the average displacement is $0.84$ meter. Then, the rupture rate is calculated where the fault slip-rate $\dot {\bar {u}}$ is 0.01 m/year

In [98]:
rupture_rate = fault_slip_rate / u_bar
rupture_rate

0.011873368076196501

In [99]:
recurrence_interval = 1/rupture_rate
recurrence_interval

84.22210055163545

This rupture rate is equivalent to $1/\lambda$ or return period or recurrence interval of $\approx 84$ years. As we know that, the fault occurrence rate given magnitude of $6.7$ is $0.02$ or equivalent to $50$ years of recurrence interval. The seismic moment approach gives the rupture rate approximately twice than the observe rate.

## Wells-Coppersmith's Relationship

The Wells-Coppersmith's (1994) has formulated the empericial equation to calculate the average displacement for the strike-slip fault mechanism as follow

$log_{10} AD = -6.32 + 0.9 M$

This equation has the standard deviation of $\pm 0.28$

In [100]:
u_bar_wc94 = 10**(-6.32 + (0.9*magnitude))
u_bar_wc94

0.5128613839913648

Based on this calculation, the average displacement is about $0.5$ meter. Then, We can calculate the rupture rate

In [101]:
rupture_rate_wc94 = fault_slip_rate / u_bar_wc94
rupture_rate_wc94

0.019498445997580455

In [102]:
recurrence_interval_wc94 = 1/rupture_rate_wc94
recurrence_interval_wc94

51.28613839913648

This value is equivalent to recurrence interval or return period $RP = 1/\lambda$ of 51 years. The deviation from the observe recurrence interval is about 1 year. So, the Wells-Coppersmith's approximation is quite close to the observation. We also can conclude that the earthquake with magnitude 6.7 has occurrence rate of $\approx 0.02$ by using Wells-Coppersmith's approach and for those fault characterizations that have been mentioned above.